Week_04/Lecture_7/assignment2/src/main/java/org/assignment1/assignment2/Services/EmployeeService.java

```diff
@@ -0,0 +1,38 @@
+package org.assignment1.assignment2.Services;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+
+@Setter
+@Getter
+
+@Service
+public class EmployeeService {
+    private String email;
+
+    // Constructor injection with @Qualifier
+    private final EmailService emailServiceConstructor;
+
+    @Autowired
```

Comment:
Can we remove this annotation? Why?

Reply:
It's a mistake and I already fix this in new commit, Thank you Ms for the feedback.

Reply:
it's still good to have this annotation, but it's not necessary - it's better if you can research more about the reason